### PR TITLE
Language and Grammar revisions

### PR DESCRIPTION
**Pull request by siemensrachel**
Swept through to clean up grammar and language issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ Click [Here](http://oscc.io) to order your autonomy kit now.
 Building and Installing Arduino Firmware
 ------------
 
-The OSCC project is built around a number of individual modules that interoperate to create a fully controllable vehicle. These modules are built from Arduinos and Arduino shields designed specifically for interfacing with various vehicle components. Once these modules have been programmed with the accompanying firmware and installed into the vehicle, the vehicle is ready to receive control commands sent over a CAN bus from a computer running a control program. 
+The OSCC Project is built around a number of individual modules that interoperate to create a fully controllable vehicle. These modules are built from Arduinos and Arduino shields designed specifically for interfacing with various vehicle components. Once these modules have been programmed with the accompanying firmware and installed into the vehicle, the vehicle is ready to receive control commands sent over a CAN bus from a computer running a control program. 
 
-**Pre-requisites:** You must already have arduino core installed on your machine.
+**Pre-requisites:** You must have Arduino Core pre-installed on your machine.
```
Comment (dan-fern) [inline README.md]: this should probably read `arduino-core` since it's an `apt-get` package

Comment (dan-fern) [inline README.md]: This should probably read `arduino-core` since it's an `apt-get` package.

Comment (ltbj) [inline README.md]: That's specifically called out in the next line when the package is installed.

Comment (dan-fern) [inline README.md]: Good call

**Pull request merged**